### PR TITLE
Initial commit for sales opportunities

### PR DESCRIPTION
**Pull request by amancdk**
---
Please check the following conditions for your PR.

- [ ] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ ] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

File: custom-connectors/EleadSalesOpportunities/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,3912 @@
...
+    "contact": {
+      "name": "CDK Global",
+      "url": "https://serviceconnect.support.cdk.com",
+      "email": "mbd_support@cdk.com"
+    }
+  },
+  "x-ms-connector-metadata": [
+      {
+        "propertyName": "Website",
+        "propertyValue": "https://fortellis.io"
```
Comment (sriyen-msft) [inline custom-connectors/EleadSalesOpportunities/apiDefinition.swagger.json:16]: @amancdk Can you please share a web page or location from your company web site where I can find some information about this service or connector.

Comment (amancdk) [reply custom-connectors/EleadSalesOpportunities/apiDefinition.swagger.json:16]: Hi @sriyen-msft , Added details in readme.md about the details. More details can be found at - https://apidocs.fortellis.io/specs/3c6d6b40-7215-4a81-a087-4f956bb36ec9

File: custom-connectors/EleadSalesOpportunities/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#1F1A3C",
+    "capabilities": [],
+    "publisher": "Bhargava, Aman Avinash"
```
Comment (dougelliott) [inline custom-connectors/EleadSalesOpportunities/apiProperties.json:6]: As with the others, please add the `stackOwner` property, and remove PII in favor of using company-specific information for both stackOwner and publisher.

**Review by dougelliott — CHANGES_REQUESTED** [review]
Please also move this to the `certified-connectors` directory, rather than `custom-connectors`

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@amancdk Please remove the icon from PR and move the folder to `certified-connectors` folder

**Comment by amancdk** [comment]
@sriyen-msft & @dougelliott made suggested changes. Thanks

**Review by dougelliott — APPROVED** [review]

**Pull request merged**